flang/lib/Parser/unparse.cpp

```diff
@@ -1847,6 +1847,8 @@ class UnparseVisitor {
             [&](const std::list<CompilerDirective::NameValue> &names) {
               Walk("!DIR$ ", names, " ");
             },
+            [&](const CompilerDirective::Error &) { Word("!DIR$ ERROR"); },
```

Comment:
How will you test this?  The directive will emit a fatal error in semantics, and then unpausing won't be run.